data/i18n/en/en.toml

```diff
@@ -234,6 +234,15 @@ other = "(released: "
 [release_date_format]
 other = "2006-01-02"
 
+[release_complete]
```

Comment:
I'd call this `release_full_details_initial_text`. We shouldn't name it after the English word.

Reply:
In my PR clone of this one, I've _added_ an entry for `release_full_details_initial_text`. I'll deprecate the `release_complete` entry and advise the other localizations to update their `<lang>/<lang>.toml` files. Eventually, after a few releases, we'll remove the `release_complete` entry.

Reply:
Oh, sorry for missed this suggestion, I have changed accordingly.